### DISCUSSION TITLE
Drive_State.Power Codes (1, 11, 21, 4, -9)?

### DISCUSSION THREAD

**Discussion by evogelpohl** [timdorr/tesla-api#366] [General]
I'm getting these values for: drive_state.power from the api, (1, 11, 21, 4, -9). 1 matches to D(drive). Others? Decoder ring?
Thanks.
'21/ModelY/Perf/FSD

**Comment by bebop4jim** [comment]
Those values are the number of kilowatts being drawn from the battery (positive values) or regenned/recharged into the battery (negative numbers).
